cea/scripts.yml

```
@@ -55,21 +55,21 @@ Solar Radiation Analysis:
       - [get_terrain]
       - [get_weather_file]
 
-  - name: radiation-usr
-    label: Building Solar Radiation using USR [BETA]
+  - name: radiation-crax
+    label: Building Solar Radiation using CRAX [BETA]
     description: |
-      **This is currently in BETA and may not work as expected.**  
-      **Only Windows are supported for now.**
+      **This is currently in active development and may not work as expected.**  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)